tools/distrib/python/grpcio_tools/grpc/tools/command.py

```diff
@@ -35,6 +35,25 @@
 from grpc.tools import protoc
 
 
+def build_package_protos(package_root):
```

Comment:
Could we either rename this to `build_proto_modules` or rename `BuildProtoModules` to `BuildPackageProtos` (my preference is on the former, but whatever's fine as long as the two correspond with each other)?